### PR TITLE
nft flask scheduler

### PR DESCRIPTION
**Pull request by Rishabh42**

### CODE REVIEW

File: scanner/app.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+# Define endpoint to query NFT ownership record
+@app.route('/get_nft_ownership', methods=['GET','POST'])
+def get_nft_ownership():
+    # Get parameters from request query string
+    nft_address = request.args.get('nft_address')
+    nft_id = request.args.get('nft_id')
+
+    # load environment variables
+    db_host = os.environ.get('DB_HOST')
```
Comment (claydrone) [inline scanner/app.py:60]: should use alchemy not direct db connection
and the db connection seetings should be out of the define

File: scripts/mint.py [hunk 2]
```diff
@@ -33,47 +32,46 @@
...
-web3.eth.wait_for_transaction_receipt(tx_hash)
-hash = web3.toHex(tx_hash)
+signed_tx = web3.eth.account.sign_transaction(mint_tx, private_key)
+tx_hash = web3.eth.send_raw_transaction(signed_tx.rawTransaction)
+receipt = web3.eth.wait_for_transaction_receipt(tx_hash)
+hash = web3.to_hex(tx_hash)
 
-print(f"Mint is completed. Tx Hash: https://hyperspace.filfox.info/en/message/{hash}")
\ No newline at end of file
+print(f"Mint is completed. Transaction Hash: {hash}")
```
Comment (claydrone) [inline scripts/mint.py:76]: using 'log.info' instead of 'printf'

**Review by claydrone — APPROVED** [review]

File: scanner/app.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+def scheduled_task():
+    print(f"Scheduled task executed at {datetime.now()}")
+
+scheduler = BackgroundScheduler()
+scheduler.add_job(func=scheduled_task, trigger="interval", seconds=10)
+scheduler.start()
+
+@app.route('/')
+def hello():
+    return jsonify(message='Hello from Flask scheduler service!')
```
Comment (flyworker) [inline scanner/app.py:27]: remove "hello world"

File: scanner/db/NFT_SQL_COMMANDS.sql [hunk 4]
```diff
@@ -34,5 +35,5 @@ SET SQL_MODE=@OLD_SQL_MODE;
 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS;
 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS;
 
-insert into nft_ownership(transfer_event_block,nft_address,nft_ID,owner_address) VALUES (34492518,'0xD81288579c13e26F621840B66aE16af1460ebB5a',2,'0xA878795d2C93985444f1e2A077FA324d59C759b0');
-insert into nft_ownership(transfer_event_block,nft_address,nft_ID,owner_address) VALUES (34492518,'0x923AfAdE5d2c600b8650334af60D6403642c1bce',2,'0xc17ae0520803E715D020C03D29D452520D6aEbf9');
\ No newline at end of file
+insert into nft_ownership(last_scan_block,transfer_event_block,nft_address,nft_ID,owner_address) VALUES (34494385,34492518,'0xD81288579c13e26F621840B66aE16af1460ebB5a',2,'0xA878795d2C93985444f1e2A077FA324d59C759b0');
+insert into nft_ownership(last_scan_block,transfer_event_block,nft_address,nft_ID,owner_address) VALUES (34494385,34492518,'0x923AfAdE5d2c600b8650334af60D6403642c1bce',2,'0xc17ae0520803E715D020C03D29D452520D6aEbf9');
```
Comment (flyworker) [inline scanner/db/NFT_SQL_COMMANDS.sql:39]: it should be object with alchemy

**Review by flyworker — APPROVED** [review]

**Comment by Rishabh42** [comment]
I have made the suggested changes to the code
Will merge this PR and open another one to the `main` branch

**Pull request merged**